### ISSUE TITLE
Shipping, End of Day process is missing

### ISSUE THREAD

**Issue by davidonelson** [DynEntTech/CBKnownIssues#43]
We need a report of packages shipped that can be provided to the carrier and/or confirm the correct boxes are being given to the carrier.

**Comment by connectedbusiness-gary** [comment]
Our solution uses the UPS Online Tools, so this report is not needed.

See...

https://support.shipworks.com/hc/en-us/articles/4407010337691-UPS-Create-and-Print-an-End-of-Day-Report
https://help.skulabs.com/en/articles/2308780-how-can-i-generate-an-end-of-day-manifest-for-ups

**Comment by davidonelson** [comment]
Agreed that a carrier like UPS does not need the reports.  The need is for the CB user.  

Customers converting from other shipping software are asking for the "end of day" process and consider it missing, so I want to have a clear answer/replacement to give them.

PART 1: 
The summary report is wanted by the CB user as an audit (a) to confirm the proper number of packages are given to each driver (b) to have the driver sign to confirm the quantity of the pickup.

PART 2:
Some businesses use the quantity/value shipped each day as a Key Performance Indicator (KPI) and are using the "end of day" report for this.  Need to figure out a clear answer/replacement for this.

**Comment by connectedbusiness-gary** [comment]
Hi,

We include a full report writer, business intelligence tools and a dashboard designer (perfect for creating KPI's) that clients / partners can use to get any required data. If the client does not want to pay a couple hundred bucks to have the report created for them, it must not be very key to their business. 

The same data can be obtained in the shipping->completed shipments screen.  As you know, we also allow the search form to be used as an "ad-hoc" report designer complete with grouping, column customization, searching and printing capabilities. for example - 6 shipments went out...

![image](https://user-images.githubusercontent.com/52012844/151115240-855d0d68-ce28-4701-9d75-b75288d982f5.png)

...now grouped by service...

![image](https://user-images.githubusercontent.com/52012844/151115393-880f8e4f-6dcf-43eb-aaca-a9f63a70288a.png)
